index.html

```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="the-rest">
+      <section class="about">
+        <div class="gap"></div>
+        <img
+          src="Images/illustration.png"
+          alt="illustration"
+          class="illustration"
+        />
```

Comment:
[REQUIRED] Kindly ensure all images that are just for decoration use CSS background property not `img` tags